src/index.ts

```
@@ -0,0 +1,402 @@
+    };
+  }
+
+  /**
+   * Creates a legacy logger for Serverless v3 compatibility
+   * @returns {Object}
+   * @private
+   */
+  private _createLegacyLogger(): Logger {
+    return {
+      info: (message: string) => {
+        if (this.serverless.cli && this.serverless.cli.log) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)